### PR TITLE
Better error handling, centralize dyanomdb conns and tables

### PR DESCRIPTION
**Pull request by loicgasser**

### CODE REVIEW

File: chsdi/views/files.py [hunk 1]
```diff
@@ -22,37 +16,12 @@
...
-            table = Table(DYNAMODB_TABLE_NAME, connection=connect_to_region('eu-west-1', debug=debugval))
-        except Exception as e:
-            raise exc.HTTPInternalServerError('Unable to access dynamodb table (%s)' % e)
-
-    return table
-
-
 def _add_item(id, file_id=False):
-    table = _get_dynamodb_table()
+    table = get_dynamodb_table('geoadmin-file-storage')
```
Comment (procrastinatio) [inline chsdi/views/files.py]: should by `get_dynamodb_table(table='geoadmin-file-storage')`

File: chsdi/views/files.py [hunk 2]
```diff
@@ -62,12 +31,12 @@ def _add_item(id, file_id=False):
...
         )
     except Exception as e:
-            raise exc.HTTPBadRequest('Error during put item %s' % e)
+        raise exc.HTTPBadRequest('Error during put item %s' % e)
     return True
 
 
 def _save_item(admin_id, file_id=None, last_updated=None):
-    table = _get_dynamodb_table()
+    table = get_dynamodb_table('geoadmin-file-storage')
```
Comment (procrastinatio) [inline chsdi/views/files.py]: should by `get_dynamodb_table(table='geoadmin-file-storage')`

File: chsdi/views/files.py [hunk 3]
```diff
@@ -100,7 +69,7 @@ def _save_item(admin_id, file_id=None, last_updated=None):
 
 
 def _is_admin_id(admin_id):
-    table = _get_dynamodb_table()
+    table = get_dynamodb_table('geoadmin-file-storage')
```
Comment (procrastinatio) [inline chsdi/views/files.py]: idem should by `get_dynamodb_table(table='geoadmin-file-storage')`

File: chsdi/views/files.py [hunk 4]
```diff
@@ -111,7 +80,7 @@ def _is_admin_id(admin_id):
 
 def _get_file_id_from_admin_id(admin_id):
     fileId = None
-    table = _get_dynamodb_table()
+    table = get_dynamodb_table('geoadmin-file-storage')
```
Comment (procrastinatio) [inline chsdi/views/files.py]: idem, or change the signaure of should by `get_dynamodb_table` !

File: chsdi/views/shortener.py [hunk 5]
```diff
@@ -76,7 +74,7 @@ def shorten_redirect(request):
     url_short = request.matchdict.get('id')
     if url_short is None:
         raise exc.HTTPBadRequest('Please provide an id')
-    table = get_table()
+    table = get_dynamodb_table_simple('shorturl')
```
Comment (procrastinatio) [inline chsdi/views/shortener.py]: again...

File: chsdi/views/files.py (hunk 2 above)
Comment (loicgasser) [inline chsdi/views/files.py]: yep

**Comment by gjn** [comment]
We can close this one, right?

**Pull request closed without merging**